src/index.js

```diff
@@ -1,150 +1,140 @@
-let allCityDetails
+import { months } from './data.js';
```

Comment:
![image](https://github.com/Abishek666/abishek_WebTips_Assignments/assets/142198666/9556c171-77be-4c8d-932e-1263b060f773)
The selected city name by default is not proper, could you update it?